notices.json

```diff
@@ -1,4 +1,23 @@
 [
+    {
+    "id": "unsupported-server-version-v5.37",
+    "conditions": {
+      "audience": "sysadmin",
+      "clientType": "all",
+      "instanceType": "onprem",
+      "repeatable": "true",
+      "serverVersion": ["<6.3"]
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "Upgrade your Mattermost server now",
```

Comment:
@nab-77 @amyblais thoughts on being more aggressive with this copy? We are targeting versions we know are unsupported here, I think it's worth flagging to the admin IMO

Reply:
Tweaked the copy, let me know feedback!

Reply:
@amyblais thanks. please coordinate with @nab-77 on copy. 

I’m in support of an aggressive message to drive upgrades but we’d need to have support trained and ready for a flood of tickets around upgrades.